### ISSUE TITLE
Plugin not picking up configuration from Star Citizen

### ISSUE THREAD

**Issue by psyko-gh** [mhwlng/streamdeck-starcitizen#11]
Hello,

**Versions:**
Star citizen: 3.17.4-LIVE.8288900
Streamdeck plugin: 1.0.8

I've been using this plugin for a few months now and it worked great.
I came back into the game a few days ago, but now I'm having trouble with it: the plugin does not see any change I do in my ingame keybindings. I have tried the following:

1. Check plugin log: no error (see log below)
2. Checked appsettings.config. Both `SCData_p4k` and `SCClientProfilePath` are good (link to actual path, no weird character in path)
3. Cleaned the .csj archives and relaunch Stream Deck as Admin: `defaultProfile.xml` get created again with default keybindings _(looks normal given the name of the file_)
4. Restarted Stream deck as Admin

But nothing works.
One thing I noticed is whenever I change a keybinding in Star Citizen, my `actionmaps.xml` is no longer updated.
The plugin correctly picks keybindings from `actionmaps.xml`, but they not longer reflect the keybinding in Star Citizen

Any chance that might be related to the Star Citzen version that changed the handling of keybindings ?

Thank you

```
2022-11-27 12:45:40.4428|DEBUG|com.mhwlng.starcitizen|1|Logger Initialized
2022-11-27 12:45:40.4648|INFO|com.mhwlng.starcitizen|1|Init Star Citizen
2022-11-27 12:45:40.4715|INFO|com.mhwlng.starcitizen|1|D:\Jeux\StarCitizen\StarCitizen\LIVE\Data.p4k
2022-11-27 12:45:49.1216|INFO|com.mhwlng.starcitizen|1|D:\Jeux\StarCitizen\StarCitizen\LIVE\USER\Client\0\Profiles\default\actionmaps.xml
2022-11-27 12:45:49.1466|INFO|com.mhwlng.starcitizen|1|Keyboard Detected, language : 40C keyboard : 40C culture : fr-FR
2022-11-27 12:45:49.1616|INFO|com.mhwlng.starcitizen|1|Keyboard Detected, language : 40C keyboard : 40C culture : fr-FR
2022-11-27 12:45:49.1846|INFO|com.mhwlng.starcitizen|1|monitoring key binding file
2022-11-27 12:45:49.1876|INFO|com.mhwlng.starcitizen|1|Finished Init Star Citizen
2022-11-27 12:45:49.1876|INFO|com.mhwlng.starcitizen|1|Plugin [com.mhwlng.starcitizen] Loading - 2 Actions Found
2022-11-27 12:45:49.3876|INFO|com.mhwlng.starcitizen|1|Plugin version: 1.0.8
2022-11-27 12:45:49.3906|INFO|com.mhwlng.starcitizen|1|Connecting to Stream Deck
2022-11-27 12:45:51.4460|INFO|com.mhwlng.starcitizen|1|Connected to Stream Deck
2022-11-27 12:45:51.4460|INFO|com.mhwlng.starcitizen|1|GlobalSettingsManager initialized
```

**Comment by mhwlng** [comment]
I checked 3.17.4 and my actionmaps.xml still changes normally when I edit a key binding in SC.

It's in a different directory from yours:

C:\Program Files\Roberts Space Industries\StarCitizen\LIVE\USER\Client\0\Profiles\default\actionmaps.xml

I guess that on your system SC is now writing actionmaps in another directory instead of D:\Jeux\StarCitizen\StarCitizen\LIVE\USER\Client\0\Profiles\default\actionmaps.xml

If you find where it is, you can edit appsettings.config with the correct path.

I don't see what I could change in the plugin to fix your problem?

**Comment by psyko-gh** [comment]
Ok I figured it out thank to your confirmation that it work on your side.
I search my drive for another actionmaps.xml and found one.
I had to reset my launcher settings recently, and did a file verification after that. I guess the game just installed into a new folder...
I guess the Launcher reste kind of messed up the game folders...
Old path:
`D:\Jeux\StarCitizen\StarCitizen\LIVE\USER\Client\0\Profiles\default\actionmaps.xml`

New path:
`D:\Jeux\StarCitizen\StarCitizen\Launcher\LIVE\USER\Client\0\Profiles\default\actionmaps.xml`

Anyway, thank you for the quick answer
